app/configurator/components/Stack.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Box, BoxProps } from "theme-ui";
+
+const Stack = ({
```

Comment:
Nit: Should we move this to app/components instead since it's a pretty generic component?

Reply:
Thanks, I am doing that in the step 2 PR :+1: